BE/src/main/java/kr/codesquad/sidedish/controller/ItemController.java

```diff
@@ -17,18 +18,31 @@
 public class ItemController {
 
     private final ItemService itemService;
+    private final OrderService orderService;
 
-    public ItemController(ItemService itemService) {
+    public ItemController(ItemService itemService, OrderService orderService) {
         this.itemService = itemService;
+        this.orderService = orderService;
     }
 
     @GetMapping
-    public ResponseEntity<List<ItemResponseDto>> findItemByCategoryId(@RequestParam Long category_id){
-        return ResponseEntity.ok(itemService.findItemByCategoryId(category_id));
+    public ResponseEntity<List<ItemResponseDto>> findItemByCategoryId(@RequestParam Long categoryId) {
+        return ResponseEntity.ok(itemService.findItemByCategoryId(categoryId));
     }
 
-    @GetMapping("/best")
-    public List<ItemListResponseDto> getItemByBestId (@RequestParam int best_id) {
-        return itemService.findByBestId(best_id);
+    @GetMapping("/event")
+    public List<ItemListResponseDto> getItemByBestId(@RequestParam int bestId) {
+        return itemService.findByBestId(bestId);
+    }
+
+    @GetMapping("/detail/{id}")
+    public ItemDetailResponseDto getItemDetailById (@PathVariable Long id) {
+        return itemService.getItemDetailById(id);
```

Comment:
여기저기서 사용되는 id가 많아진다면 이런식으로도 활용 할 수 있습니다.

```suggestion
    @GetMapping("/detail/{id}")
    public ItemDetailResponseDto getItemDetailById (@PathVariable(name = "id") Long itemId) {
        return itemService.getItemDetailById(itemId);
```

Reply:
앗 해당 사항 정리해두겠습니다. 감사합니다!